internal/controller/driver_controller.go

```diff
@@ -757,6 +758,7 @@ func (r *driverReconcile) reconcileControllerPluginDeployment() error {
 							})
 						}
 						// Addons Sidecar Container
+						var port = r.ControllerPluginCsiAddonsContainerPort()
```

Comment:
```suggestion
port := r.ControllerPluginCsiAddonsContainerPort()
```

Lets use a short declaration and move it inside the `if !r.isNfsDriver() && ptr.Deref(r.driver.Spec.DeployCsiAddons, false) {` check as its required only when csiaddons is enabled